public/app/features/alerting/partials/alert_tab.html

```diff
@@ -29,9 +33,14 @@ <h4 class="section-heading">Rule</h4>
               been firing for more than For duration, it will change to Alerting and send alert
               notifications.
             </info-popover>
-          </div>
         </div>
       </div>
+      <div class="gf-form" ng-if="ctrl.frequencyWarning">
+        <label class="gf-form-label text-warning">
+          <i
```

Comment:
Is `<i></i>` an icon with Angular? Don't remember, otherwise it would be italic text?

Reply:
I have just copied it from [above](https://github.com/grafana/grafana/blob/a6a7e0d2ac083dc9295bd7821209c1174ad13bab/public/app/features/alerting/partials/alert_tab.html#L3).

Reply:
@papagian Aha OK thanks, it's probably an icon since it doesn't contain any text.